### ISSUE TITLE
Staging Review finding: Review "Edit" buttons cannot be distinguished from one another when using a screen reader

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104801]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/ask-va/review-then-submit](https://staging.va.gov/contact-us/ask-va/review-then-submit), each editable section has an "Edit" button. In order to provide a unique accessible name to make it easier for screen reader users to distinguish between these buttons, we often include `aria-label` text on these kinds of repeated elements.

As currently coded:
- Some Edit buttons do not have an `aria-label`, meaning a screen reader user may not be sure what they're attempting to edit
- Some Edit button have identical `aria-label` text, `Edit Your school information`, which does not match what's actually being edited (appears to be copy/pasted)

## Recommended action

Ensure that each Edit button has unique `aria-label` that accurately describes its function and allows users to distinguish it from other Edit buttons.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by arielma-tw** [comment]
Issue addressed withhttps://github.com/department-of-veterans-affairs/ask-va/issues/1659
